packages/bpk-component-map/src/BpkMap.js

```diff
@@ -114,27 +146,30 @@ const BpkMap = withGoogleMap((props: Props) => {
         streetViewControl: false,
         fullscreenControl: false,
         rotateControl: false,
+        clickableIcons: false,
+        styles: mapOptionStyles,
```

Comment:
It can support us to hide the hotels icons on the maps after adding some custom markers on the map to avoid the conflicts
![image](https://user-images.githubusercontent.com/29422130/165711718-68594014-22a4-48ed-8d5c-c1e5e01ac10e.png)